tests/test_declarative_pipeline.py

```
@@ -1055,6 +1055,45 @@ def test_mounts_extra_keep_mounts_true_additive_merge(self):
+        assert exec_config["keep_mounts"] is False
+
+        pipeline = object.__new__(Pipeline)
+        pipeline.with_ray = False
+        hardware = HardwareConfig(num_gpus=0, num_nodes=1, num_tasks=1)
+        with patch("nemo_skills.pipeline.utils.declarative.get_executor") as mock_get_executor:
+            pipeline._create_executor(
+                cmd,
+                exec_config,
+                "sandbox:latest",
+                cluster_config,
+                "/tmp/logs",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/logs"

(S108)